modules/arm_plugin/src/opset/pool_arm.hpp

```diff
@@ -56,7 +56,7 @@ class ArmAvgPool : public ov::op::v1::AvgPool {
 namespace v8 {
 class ArmMaxPool : public ov::op::v8::MaxPool {
 public:
-    OPENVINO_OP("ArmMaxPool", "arm_opset", ov::op::v8::MaxPool, 8);
+    OPENVINO_OP("ArmMaxPool", "arm_opset", ov::op::v8::MaxPool);
```

Comment:
```suggestion
    OPENVINO_OP("ArmMaxPool", "arm_opset8", ov::op::v8::MaxPool);
```
Without changes in opset you will get collisions between these ops.